app/views/tas/_upload_modal.html.erb

```diff
@@ -0,0 +1,33 @@
+<%= content_for :modal_id, 'upload_dialog' %>
+<%= content_for :modal_title, I18n.t('upload') %>
```

Comment:
In views, use `t`, not `I18n.t`. (http://api.rubyonrails.org/classes/ActionView/Helpers/TranslationHelper.html#method-i-translate)